Logger.Tests/BookTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Logger.Tests;
+
+[TestClass]
+public class BookTests
+{
+    private readonly string testTitle = "Of Mice and Men";
+    private readonly FullName testAuthor = new(FirstName: "John", LastName: "Steinbeck", MiddleName: "Ernst");
+    private readonly int testYearPublished = 1937;
+
+    [TestMethod]
+    public void Book_SetsProperties_Success()
+    {
+        Book testBook = new(testTitle, testAuthor, testYearPublished);
+
+        Assert.AreEqual(testBook.Title, "Of Mice and Men");
+        Assert.AreEqual(testBook.Author.ToString(), "John Ernst Steinbeck");
+        Assert.AreEqual(testBook.YearPublished, 1937);
+    }
```

Comment:
I think best practice would be to only use one Assert per test. Otherwise if the test fails, which property failed? Which property are you testing? 
I feel like a better way of doing it would be to just simplify it and if an exception is thrown you'll know that the object creation failed. 

```suggestion
    [TestMethod]
    public void Book_SetsProperties_Success()
    {
        Book testBook = new(testTitle, testAuthor, testYearPublished);
    }
```

Reply:
VS will tell you which specific Assert statements failed, what it expected from the statement and what was actually returned. I am kind of confused on what you're suggesting here, as there are no Assert statements in the above suggested change.

Reply:
I think you should leave it honestly. I feel like I just got too caught up on him saying in class that we shouldn't test multiple things in one method, but I mean you're testing book creation which is technically one thing. 
The code suggestion I made is similar to some of the tests the professor made in class. It would just test that there is no compile error I guess. I agree with what you are saying though, it doesn't really make that much sense because there are no asserts or anything. I would leave your code as is, I think I'm wrong about this one.